packages/react/src/utils/validateRuleContrib.ts

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+import { validate } from "compare-versions";
+
+// Use error handling
+export const checkPackageExists = async (packageName: string, packageVersion?: string) => {
+  let query = `https://azuresearch-usnc.nuget.org/query?q=${packageName}`;
+  if (packageVersion) {
+    query = query + `&semVerLevel=${packageVersion}`;
+  }
+  const response = await axios.get(query);
+  if (response.status !== 200) {
+    throw new Error(
+      `Unable to query for provided package. URL: ${query}, Response: ${response.status} - ${response.statusText}`
```

Comment:
do not throw error here, I remember we do not throw any error in UI. It may cause issue. Make sure you have try catch to handle it. otherwiese just use some error message

Reply:
I have a try catch statement to handle, and I show a flashbar indicating to the user that some error occurred. is that okay, or should I not throw the error at all?